aspnetcore/tutorials/web-api-help-pages-using-swagger/samples/2.0/TodoApi.Swashbuckle/TodoApi.csproj

```diff
@@ -5,6 +5,12 @@
     <ProjectUISubcaption>Swashbuckle - ASP.NET Core 2.0</ProjectUISubcaption>
   </PropertyGroup>
 
+  <!-- <snippet_NoDocumentGeneration> -->
```

Comment:
Another snippet that you probably meant to import into the doc.

Reply:
No, another bit of prep